aws.yaml

```diff
@@ -1,18 +1,18 @@
-- active: false
+- active: true
   authorities:
     - name: app-operator
       version: 1.0.0
     - name: aws-operator
-      version: 7.0.1-dev
+      version: 8.0.0
     - name: cert-operator
       version: 0.1.0
     - name: chart-operator
       version: 0.7.0
     - name: cluster-operator
       provider: aws
-      version: 2.0.0-dev
-  date: 2019-12-05T12:00:00Z
-  version: 10.0.0
+      version: 2.0.0
+  date: 2019-12-18T14:00:00Z
+  version: 10.1.0
 - active: false
```

Comment:
You'll also need to add a new release with aws-operator@8.0.1-dev and cluster-operator@2.0.1-dev for testing. Do you want to provide that in a separate PR?

Reply:
We can add this here before merging.